code/test/CoreTemplateStudio.Core.Test/Extensions/StringExtensionsTests.cs

```diff
@@ -10,11 +10,55 @@ namespace Microsoft.Templates.Core.Test.Extensions
     [Trait("ExecutionSet", "Minimum")]
     public class StringExtensionsTests
     {
+        [Theory]
+        [InlineData("string")]
+        [InlineData("string23TEST")]
+        public void ObfuscateSHA_ShouldNotBeEmpty(string value)
+        {
+            var actual = value.ObfuscateSHA();
+
+            Assert.NotEmpty(actual);
+        }
+
+        [Theory]
+        [InlineData("string")]
+        [InlineData("string23TEST")]
+        public void ObfuscateSHA_ShouldNotBeEqualsToTheValue(string value)
+        {
+            var actual = value.ObfuscateSHA();
+
+            Assert.False(value.Equals(actual, StringComparison.Ordinal));
+        }
+
+        [Theory]
+        [InlineData("string")]
+        [InlineData("string23TEST")]
+        public void ObfuscateSHA_ShouldNotContainLowercase(string value)
+        {
+            var actual = value.ObfuscateSHA();
+
+            Assert.DoesNotContain(actual, c => char.IsLower(c));
+        }
+
+        [Theory(Skip = "Control null and empty scenarios")]
```

Comment:
Is there a follow-up task being created to expand upon this test and address why it is being skipped?
Adding this test raises lots of questions and it's not clear what is meant by "control these scenarios".

Reply:
Thanks for all the feedback provided Matt. It´s really helpful. 
I am working on this task to help the team to increase the code coverage in CoreTS, and if possible improving the code itself. I am not too familiar with the code of CoreTemplateStudio myself, that´s why the PR is in draft. This way, I can get feedback from the Team while adding some tests and confirm that the tests are adding some real value in here. 

Depending of what other missing scenarios the code could have, we are going to create a follow-up task or fix those scenarios at the end in this same PR.